packages/razorsharp/src/code/blade/components/alert/alert.ts

```diff
@@ -0,0 +1,97 @@
+import { jsxValue } from '../../utils/attributes';
+import { component } from '../../utils/component';
+import { isPresent } from '../../utils/isPresent';
+import { findNode } from '../../utils/findNode';
+import { findTextByLayerName } from '../../utils/findTextByLayerName';
+import { bladeImports } from '../../utils/imports';
+import { defaultValues } from './constants';
+import type { TransformFunctionReturnType } from '~/code/types/TransformFunction';
+import type { BladeComponentInstanceNode, BladeProps, BladeTextNode } from '~/code/types/Blade';
+
+export const transformAlert = (
+  bladeInstance: BladeComponentInstanceNode,
+): TransformFunctionReturnType => {
+  const componentProperties = bladeInstance.componentProperties;
+
+  const intent = jsxValue(componentProperties.Intent?.value).toLowerCase();
+
+  const contrast = jsxValue(componentProperties.contrast?.value).toLowerCase();
+
+  const isDismissible = jsxValue(componentProperties.isDismissible?.value).toLowerCase();
+
+  const isFullWidth = jsxValue(componentProperties.isFullWidth?.value).toLowerCase();
+
+  const title = findTextByLayerName(bladeInstance, 'title') ?? '';
+  const description = findTextByLayerName(bladeInstance, 'message') ?? '';
+
+  const props: BladeProps = {
+    intent: {
+      type: 'string',
+      value: intent,
+    },
+    contrast: {
+      type: 'string',
+      value: contrast,
+    },
+    isDismissible: {
+      type: 'boolean',
+      value: isDismissible,
+    },
+    isFullWidth: {
+      type: 'boolean',
+      value: isFullWidth,
+    },
+    title: {
+      type: 'string',
+      value: title,
+    },
+    description: {
+      type: 'string',
+      value: description,
+    },
+  };
+
+  const isPrimaryActionPresent = isPresent(componentProperties.primaryAction?.value);
+  const isSecondaryActionPresent = isPresent(componentProperties.secondaryAction?.value);
+
+  let actions = '{';
+
+  if (isPrimaryActionPresent) {
+    const primaryActionBaseWrapper = findNode(
+      bladeInstance,
+      (node) => node.type === 'INSTANCE' && node.layerName === '_CButton',
+    ) as BladeComponentInstanceNode;
```

Comment:
findNode returns BladeNode | null, then we are casting it too, make sure we are not missing any null checks